### DISCUSSION TITLE
Blocked domains from GitHub Actions

### DISCUSSION THREAD

**Discussion by rachelcarmena** [community/community#25068] [Actions]
<p>We’ve been able to get content from <strong>bintray.com</strong> from GitHub actions so far (daily execution).</p>
<p>However, today, we got an error because it’s not possible to get this file:</p>
<pre><code>curl https://dl.bintray.com/kotlin/kotlin-dev/org/jetbrains/kotlin/kotlin-compiler/maven-metadata.xml
</code></pre>
<p>The result is:</p>
<pre><code>% Total % Received % Xferd Average Speed Time Time Time Current
                                 Dload Upload Total Spent Left Speed

  0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0
  0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0
  0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0
</code></pre>
<p>Please, could <strong>bintray.com</strong> be added to the <em>white list</em>?</p>
<p><strong>curl</strong> works OK to get content from other domains like <strong>maven.org</strong> or <strong>githubusercontent.com</strong>.</p>
<p>Thanks in advance!</p>

**Comment by Yanjingzhu** [comment]
<p>I tried to run curl command directly in my Ubuntu machine, it could not get any data either.</p>
<pre><code>curl https://dl.bintray.com/kotlin/kotlin-dev/org/jetbrains/kotlin/kotlin-compiler/maven-metadata.xml
</code></pre>
<p>I tried to get <a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdl.bintray.com%2Fkotlin%2Fkotlin-dev%2Forg%2Fjetbrains%2Fkotlin%2Fkotlin-compiler&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7Ce2d2420abc034b9050ea08d797e7a2e4%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637144893432886713&amp;sdata=NdfaWdvceGLp%2F9lWHvjM5gDz2dSstMURlbPJ%2B%2Flo8bY%3D&amp;reserved=0" rel="nofollow noopener">https://dl.bintray.com/kotlin/kotlin-dev/org/jetbrains/kotlin/kotlin-compiler</a> , I could get data.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/029873b6cabe13ac5c40d0d9f3bb86babdbe36dc.png" data-download-href="/uploads/short-url/mXA4FTaxMwTZeSopaWI4ZBrMGo.png?dl=1" title="curl data.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181088646-4816a497-a58a-4a4d-802b-39cbe5cc562b.png" alt="curl data.png" data-base62-sha1="mXA4FTaxMwTZeSopaWI4ZBrMGo" width="689" height="167" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/029873b6cabe13ac5c40d0d9f3bb86babdbe36dc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">curl data.png</span><span class="informations">1148×278 12.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>In my opinion,  this is not a Github Action issue. As I am not familiar with <a href="http://bintray.com" rel="nofollow noopener">bintray.com</a>, could you please check whether it supported to get maven-metadata.xml through curl ?  </p>

**Comment by rachelcarmena** [comment, marked as answer]
<p>Thanks <a class="mention" href="/yanjingzhu">@yanjingzhu</a> for your answer!! </p>
<p>It seems something was fixed because today it’s working:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b1c6cb502ce565c082ddffbd4004431cf5e8236c.png" data-download-href="/uploads/short-url/pmGqsR7HSwnnwtlhu9r92qaAYC8.png?dl=1" title="Screenshot from 2020-01-13 11-06-39.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/22792183/181088648-cc2f53d8-dd36-4947-a703-998cd7c91ac2.png" alt="Screenshot from 2020-01-13 11-06-39.png" data-base62-sha1="pmGqsR7HSwnnwtlhu9r92qaAYC8" width="690" height="167" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b1c6cb502ce565c082ddffbd4004431cf5e8236c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-01-13 11-06-39.png</span><span class="informations">928×225 26.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I pointed GitHub because it has been working for several weeks (daily execution) and it worked OK in other environments as well (I didn’t have your problem with my Ubuntu machine).</p>
<p>In any case, you also gave me a useful clue with your test. You got a 301 answer so I’ll save that info in case the error happens again.</p>
<p>Thanks again <a class="mention" href="/yanjingzhu">@yanjingzhu</a> </p>
